Rakefile

```diff
@@ -5,4 +5,5 @@ Yast::Tasks.submit_to :sle12sp1
 Yast::Tasks.configuration do |conf|
   #lets ignore license check for now
   conf.skip_license_check << /.*/
+  conf.obs_sr_project = "SUSE:SLE-12-SP1:Update"
```

Comment:
@mvidner This is intended to be release as a maintenance update (it's not a P1 or a P2) and `sle12sp1` points to `SUSE:SLE-12-SP1:GA`. Also maybe we should update `yast-rake` as soon as SP1 is finally released.